### PR TITLE
chore(e2e): missing checks and cleanup

### PR DESCRIPTION
**Pull request by emosbaugh**
Adds some missing checks to existing tests. Consolidates some code. Adds retries.

### CODE REVIEW

File: e2e/install_test.go [hunk 1]
```diff
@@ -716,6 +703,19 @@ func TestResetAndReinstallAirgap(t *testing.T) {
 		t.Fatalf("fail to install embedded-cluster on node %s: %v", tc.Nodes[0], err)
 	}
 
+	if err := setupPlaywright(t, tc); err != nil {
+		t.Fatalf("fail to setup playwright: %v", err)
+	}
+	if _, _, err := runPlaywrightTest(t, tc, "deploy-app"); err != nil {
+		t.Fatalf("fail to run playwright test deploy-app: %v", err)
+	}
```
Comment (ricardomaraschini) [inline e2e/install_test.go:711]: There is a function called `setupPlaywrightAndRunTest` that executes both things at the same time.

File: e2e/scripts/single-node-airgap-install.sh [hunk 2]
```diff
@@ -27,14 +27,34 @@ main() {
...
     fi
+    if ! ensure_version_metadata_present; then
+        echo "Failed to check the presence of the version metadata configmap"
+        exit 1
+    fi
+    if ! ensure_binary_copy; then
+        echo "Failed to ensure the embedded binary has been copied to /var/lib/embedded-cluster/bin"
+        exit 1
+    fi
+    # if ! install_kots_cli; then
```
Comment (ricardomaraschini) [inline e2e/scripts/single-node-airgap-install.sh:54]: Any reason to keep these commented sections ?

File: e2e/utils.go [hunk 3]
```diff
@@ -191,3 +191,22 @@ func runInParallel(t *testing.T, fns ...func(t *testing.T) error) {
 		}
 	}
 }
+
+func installTestDependencies(t *testing.T, tc *cluster.Output, node int, withProxy bool) {
```
Comment (ricardomaraschini) [inline e2e/utils.go:195]: This work only for debian/ubuntu no ?

File: e2e/scripts/single-node-install.sh [hunk 4]
```diff
@@ -219,6 +163,9 @@ main() {
         exit 1
     fi
 
+    echo "ensure that installation is installed"
+    kubectl get installations --no-headers | grep -q "Installed"
```
Comment (ricardomaraschini) [inline e2e/scripts/single-node-install.sh:167]: Isn't the failure of this become a little bit "mysterious" or hidden even ?

**Review by sgalsaleh — APPROVED** [review]

**Pull request merged**